support/docker/dev/janitor.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "PeerTube",
+  "description": "Federated (ActivityPub) video streaming platform using P2P (BitTorrent) directly in the web browser with WebTorrent and Angular.",
+  "icon": "https://janitor.technology/img/peertube.svg",
+  "docker": {
+    "image": "chocobozzz/peertube-dev"
+  },
+  "ports": {
+    "22": {
+      "label": "SSH",
+      "proxy": "none"
+    },
+    "8088": {
+      "label": "VNC",
+      "proxy": "https"
+    },
+    "8089": {
+      "label": "Cloud9",
+      "proxy": "https"
+    },
+    "9000": {
+      "label": "PeerTube",
+      "proxy": "https",
```

Comment:
PeerTube will be behind https? Because we would need to update the configuration: https://github.com/Chocobozzz/PeerTube/blob/develop/config/default.yaml#L5

Reply:
`"proxy": "https"` actually means that our HTTPS web proxy will first authenticate users (and verify that they are allowed to access the requested container) and then connect to PeerTube's HTTP port in the container to serve content.